src/software/new_geom/polynomial2d.h

```diff
@@ -25,14 +25,40 @@ class Polynomial2d
      */
     Polynomial2d(Polynomial1d poly_x, Polynomial1d poly_y);
 
+    /**
+     * Construct a 2d polynomial that passes through the given points
+     *
+     * The created polynomial will be at the first point at t=0, and at the last point at
+     * t=1, with the other points linearly interpolated in between; ie. if we had three
```

Comment:
Super nit:

In-between

Reply:
Replaced every instance (there were 5) in the codebase